### PR TITLE
XDR - IR: fix wrong close reason by mirror out

### PR DESCRIPTION
**Pull request by ilappe**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-14732

## Description
when closing incident in XDR -> mirror in -> the incident in XSOAR will be closed
-> mirror out -> the XDR will close again with incorrect close reason (in some cases)

### CODE REVIEW

File: Packs/CortexXDR/ReleaseNotes/4_8_5.md [hunk 1]
```diff
@@ -11,4 +11,4 @@
 - Deprecated the command ***xdr-run-script-file-exists***. Use ***xdr-file-exist-script-execute*** instead.
 - Deprecated the command ***xdr-endpoint-scan***. Use ***xdr-endpoint-scan-execute*** instead.
 - Deprecated the command ***xdr-unisolate-endpoint***. Use ***xdr-endpoint-unisolate*** instead.
-- Deprecated the command ***xdr-run-snippet-code-script***. Use ***xdr-snippet-code-script-execute*** instead.
\ No newline at end of file
+- Deprecated the command ***xdr-run-snippet-code-script***. Use ***xdr-snippet-code-script-execute*** instead.
```
Comment (yaakovi) [inline Packs/CortexXDR/ReleaseNotes/4_8_5.md:14]: EOL?

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py [hunk 2]
```diff
@@ -619,7 +620,7 @@ def test_get_remote_data_command_should_close_issue(requests_mock, mocker):
     expected_modified_incident['assigned_user_mail'] = ''
     expected_modified_incident['assigned_user_pretty_name'] = ''
     expected_modified_incident['closeReason'] = 'Resolved'
-    expected_modified_incident['closeNotes'] = 'Handled'
+    expected_modified_incident['closeNotes'] = MIRROR_IN_CLOSE_REASON + '\nHandled'
```
Comment (yaakovi) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py:623]: Use f-string instead

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 3]
```diff
@@ -750,11 +750,12 @@ def handle_incoming_closing_incident(incident_data):
             'ContentsFormat': EntryFormat.JSON
         }
         incident_data['closeReason'] = XDR_RESOLVED_STATUS_TO_XSOAR.get(incident_data.get("status"))
-        incident_data['closeNotes'] = incident_data.get('resolve_comment')
+        incident_data['closeNotes'] = MIRROR_IN_CLOSE_REASON + f'\n{incident_data.get("resolve_comment")}'
```
Comment (yaakovi) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:753]:
```suggestion
        incident_data['closeNotes'] = '{MIRROR_IN_CLOSE_REASON}\n{incident_data.get("resolve_comment")}'
```

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 4]
```diff
@@ -750,11 +750,12 @@ def handle_incoming_closing_incident(incident_data):
...
 
         if incident_data.get('status') == 'resolved_known_issue':
-            closing_entry['Contents']['closeNotes'] = 'Known Issue.\n' + incident_data['closeNotes']
-            incident_data['closeNotes'] = 'Known Issue.\n' + incident_data['closeNotes']
+            close_notes = 'Known Issue.\n' + incident_data.get('closeNotes', '')
```
Comment (yaakovi) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:756]: same

File: Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py [hunk 5]
```diff
@@ -129,6 +129,8 @@
     "tenantId",
 }
 
+MIRROR_IN_CLOSE_REASON = 'Closed during mirroring-in due to remote incident was closed'
```
Comment (yaakovi) [inline Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py:132]:
```suggestion
MIRROR_IN_CLOSE_REASON = 'Closed during mirroring-in due to the remote incident being closed.'
```

File: Packs/Core/ReleaseNotes/1_3_4.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Investigation & Response
+- Fixed an issue where *Close Reason* was incorectly changed by mirroring out.
```
Comment (yaakovi) [inline Packs/Core/ReleaseNotes/1_3_4.md:4]: There is no mirroring in XSIAM. remove RN; No need to bump version

**Comment by ilappe** [comment]
validation fail on release notes missed in Core pack for `CortexCoreIR`
no RN needed for this pack as there is no mirroring in XSIAM

**Review by yaakovi — APPROVED** [review]

**Comment by yaakovi** [comment]
validation failed on missing RN for XSIAM pack (not needed)

**Pull request merged**